packages/graphql/src/translate/cypher-builder/variables/RelationshipRef.test.ts

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as CypherBuilder from "../CypherBuilder";
+import { TestClause } from "../utils/TestClause";
+
+describe("RelationshipRef", () => {
+    it("Create Relationshipt from node", () => {
```

Comment:
A typo must have sneaked itself into the PR
```suggestion
    it("Create relationship from node", () => {
```